tests/topotests/conftest.py

```diff
@@ -160,9 +160,9 @@ def check_for_memleaks():
     if tgen is not None:
         tgen.valgrind_existing_files = existing
 
-    if leaks:
-        logger.error("valgrind memleaks found:\n\t%s", "\n\t".join(leaks))
-        pytest.fail("valgrind memleaks found for daemons: " + " ".join(daemons))
```

Comment:
This topotests change looks like a temporary hack snuck in?

Reply:
ah fuck
 I will fix

Reply:
fixed